app/models/concerns/request_validations.rb

```diff
@@ -18,11 +18,15 @@ module RequestValidations
   protected
 
   def destination_is_a_pickup_library
-    return if pickup_libraries.include?(destination)
+    return if check_destination?(destination)
 
     errors.add(:destination, 'is not a valid pickup library')
   end
 
+  def check_destination?(destination)
```

Comment:
Very minor but if you only use this method once in the codebase, potentially it could be more readable if you just directly put the check into line 21 instead of writing a method.

Reply:
will do!